src/views/FileBrowserView/FileBrowser.vue

```diff
@@ -287,21 +295,24 @@ export default {
       async get() {
         const { version, identifier, location } = this;
```

Comment:
```suggestion
        const { version, identifier, location, page } = this;
```
So you don't have to use `this.page` farther down.